thirdeye/thirdeye-frontend/app/mirage/endpoints/rootcause.js

```diff
@@ -0,0 +1,48 @@
+export default function(server) {
+  /**
+   * Get request for rootcause page
+   * @return {Array}
+   */
+  server.get(`/rootcause/raw`, () => {
+    return [ {
+      "urn" : "thirdeye:metric:1",
+      "score" : 1.0,
+      "label" : "thirdeye::pageViews",
+      "type" : "metric",
+      "link" : null,
+      "relatedEntities" : [ ],
+      "attributes" : {
+        "inverse" : [ "false" ],
+        "dataset" : [ "thirdeye" ],
+        "derived" : [ "false" ],
+        "additive" : [ "true" ]
+      }
+    } ];
+  });
+
+  /**
+   * Retrieves the options for autocomplete for 'filter' dropdown
+   */
+  server.get('/data/autocomplete/filters/metric/1', () => {
+    return {
+      environment: ['prod']
+    };
+  });
+
+  /**
+   * Retrieves information about a metric
+   */
+  server.get('/data/metric/1', () => {
+    return {
+      "id": 1,
+      "alias": "pageViews"
+    };
+  });
+
+  /**
+   * Retrieves timestamp of last element in timeseries
+   */
+  server.get('/data/maxDataTime/metricId/1', () => {
+    return 1517846399998;
```

Comment:
is it timeproof? i.e. we run this in 5 months, is it still going to work?

Reply:
For the test, the number mocked here is irrelevant for our test case right now. I might revisit this for when we test the chart. @apucher thoughts?

Reply:
not relevant at this point. subject to change once we mock timeseries data.